tabular/src/autogluon/tabular/models/widedeep_nn/preprocessing_utils.py

```diff
@@ -0,0 +1,98 @@
+import copy
+import logging
+
+import pandas as pd
+from sklearn.preprocessing import StandardScaler, MinMaxScaler
+
+from autogluon.common.features.types import R_FLOAT, R_DATETIME, S_TEXT_SPECIAL
+from autogluon.core.constants import REGRESSION, QUANTILE
+
+logger = logging.getLogger(__name__)
+
+
+class ContinuousNormalizer:
```

Comment:
Should these live in autogluon.features?

Reply:
I initially just refactored in-place, but since 100% of this code is reused in fastai models, it makes sense to put in the common place. I'll refactor fastai in other PR.